pkg/arvo/sur/invite-store.hoon

```diff
@@ -1,45 +1,40 @@
+/-  *resource
 |%
 ++  serial  @uvH
 ::
 +$  invite
   $:  =ship           ::  ship to subscribe to upon accepting invite
       app=@tas        ::  app to subscribe to upon accepting invite
-      =path           ::  path to subscribe to upon accepting invite
+      =resource       ::  path to subscribe to upon accepting invite
       recipient=ship  ::  recipient to receive invite
       text=cord       ::  text to describe the invite
   ==
 ::
-::  +invites: each application using invites creates its own path that
+::  +invites: each application using invites creates its own resource that
 ::  contains a map of serial to invite. this allows it to only receive
 ::  invites that it is concerned with
 ::
-+$  invites  (map path invitatory)   ::  main data structure
++$  invites  (map resource invitatory)   ::  main data structure
```

Comment:
The `path` here is actually `/chat`, `/contacts`, etc. I think this should actually just be changed to `term`

Reply:
Also these keys will need to be updated in the various `+on-init` where they're created.

Reply:
done